app/controllers/index.js

```diff
@@ -1,7 +1,42 @@
 import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { tracked } from '@glimmer/tracking';
+import { inject as service } from '@ember/service';
 
 export default class IndexController extends Controller {
+  @service store;
+  @service toaster;
+
   sort = '-created';
   page = 0;
   size = 20;
+
+  selectedForm;
+
+  @tracked showDeleteModal = false;
```

Comment:
Optional: it might be better to make the `selectedForm` tracked (and maybe rename it to `formToDelete`) and convert `showDeleteModal` to a getter that checks if a record is stored or not.

That way there is only 1 field that needs to be managed instead of 2.